logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/Call.kt

```diff
@@ -13,5 +14,7 @@ enum class CallStatus {
 
 data class Call(
     val conversationId: ConversationId,
-    val status: CallStatus
+    val status: CallStatus,
+    val participants: List<Participant> = emptyList(),
+    val maxParticipants: Int = 0 // Was used for tracking
```

Comment:
This particular attribute was being used for tracking in legacy app, will keep it for now as it is also easier for us to test when someone joins and leaves the call so we can compare both `participants.size` and `maxParticipants`